src/tests/unit/tests/common/configs/needs-review-rule-resources.test.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {
+    getNeedsReviewRuleResourcesUrl,
+    isOutcomeNeedsReview,
+} from 'common/configs/needs-review-rule-resources';
+
+describe(getNeedsReviewRuleResourcesUrl, () => {
+    const ruleId = 'rule-id';
+
+    it('for ruleId passed to get NeedsReviewRuleResourcesUrl', () => {
+        const needsReviewRuleResourcesPath =
+            'https://accessibilityinsights.io/info-examples/web/needs-review';
+
+        const expectedUrl = `${needsReviewRuleResourcesPath}/${ruleId}`;
+
+        const resourceUrl = getNeedsReviewRuleResourcesUrl(ruleId);
+
+        expect(resourceUrl).toBe(expectedUrl);
+    });
+});
+
+describe(isOutcomeNeedsReview, () => {
+    const ruleId = 'color-contrast';
+    const outcomeType = 'review';
+
+    it('for outcome that is review and ruleId is eligble for needsReview', () => {
+        const expectedBoolean = true;
```

Comment:
2 thoughts:
1. `expectedResult` would be a far better name than `expectedBoolean`, since it captures the intent, not just the type.
2. In a simple test like this, having everything parameterized just makes it harder to read, especially with a screen reader. There are at least 2 ways to improve this. Either (1) simply the test to something like 
```
        expect(isOutcomeNeedsReview('color-contrast', 'review')).toBe(true);
```
or (2) keep the parameters but use the it.each pattern to specify the parameters as part of an input array, similar to what is done in [this test](https://github.com/microsoft/accessibility-insights-web/blob/main/src/tests/unit/common/element-snapshot-formatter.test.ts#L14-L28)

Reply:
Refactored the test file like
        expect(isOutcomeNeedsReview('color-contrast', 'review')).toBe(true);